app/components/jobseekers/school_overview_component.html.haml

```diff
@@ -37,13 +37,20 @@
           %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
             = t('schools.website')
           %td.govuk-table__cell
-            = link_to(t('jobs.website_link_text', organisation_name: @vacancy.parent_organisation.name), @vacancy.parent_organisation.website.presence || @vacancy.parent_organisation.url, class: 'govuk-link link-wrap', target: '_blank')
+            = link_to(t('schools.website_link_text', organisation_name: @vacancy.parent_organisation.name), @vacancy.parent_organisation.website.presence || @vacancy.parent_organisation.url, class: 'govuk-link link-wrap', target: '_blank')
```

Comment:
For potential git conflict future reference: in my branch I set @school to @vacancy.parent_organisation in the school overview component, to save space/duplication. (Maybe I'll attribute reader-ify it.)